ruby-test-mode.el

```diff
@@ -69,49 +70,92 @@ Test Driven Development in Ruby."
   "The keymap used in `ruby-test-mode' buffers.")
 
 (defcustom ruby-test-file-name-extensions
-  '("builder" "erb" "haml" "rb" "rjs")
+  '("builder" "erb" "haml" "rb" "rjs" "rake" "slim")
```

Comment:
@zw963 I would say add the mappings that you are using, that you can test or verify, and leave the other mappings for someone else ;)